conda/utils.py

```diff
@@ -255,8 +255,16 @@ def find_parent_shell(path=False):
         #          "To proceed, please conda install psutil")
         return None
```

Comment:
I added this `return None` here, right above.  I'm guessing you don't want that in this function now, correct?